tests/queries/0_stateless/03036_dynamic_read_subcolumns.sh

```diff
@@ -1,62 +0,0 @@
-#!/usr/bin/env bash
-# Tags: long
-
-CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
-# reset --log_comment
-CLICKHOUSE_LOG_COMMENT=
-# shellcheck source=../shell_config.sh
-. "$CUR_DIR"/../shell_config.sh
-
-CH_CLIENT="$CLICKHOUSE_CLIENT --allow_experimental_variant_type=1 --use_variant_as_common_type=1 --allow_experimental_dynamic_type=1"
-
-
-function test()
-{
-    echo "test"
-    $CH_CLIENT -q "insert into test select number, number from numbers(100000) settings min_insert_block_size_rows=50000"
```

Comment:
As an option, you can merge all the queries inside `test()` into a single query to not duplicate the tests. But if it would still timeout then it is ok :)

Reply:
First I also thougth about it, but decided to split and rewrite everything to sql test to just forget about this problem with timeouts. I think a bit duplicated queries in tests is better then flaky tests :)